assets/js/modules/analytics/components/common/PropertySelect.test.js

```diff
@@ -95,6 +95,30 @@ describe( 'PropertySelect', () => {
 			.toHaveClass( 'mdc-select--disabled' );
 	} );
 
+	it( 'should not render if account ID is not set.', async () => {
+		const { container, registry } = render( <PropertySelect />, {
+			setupRegistry: ( { dispatch } ) => {
+				const { properties } = fixtures.accountsPropertiesProfiles;
+				const accountID = properties[ 0 ].accountId; // eslint-disable-line sitekit/acronym-case
+
+				dispatch( MODULES_TAGMANAGER ).setSettings( {} );
+				dispatch( STORE_NAME ).setSettings( {} );
+
+				dispatch( STORE_NAME ).receiveGetExistingTag( null );
+
+				dispatch( STORE_NAME ).receiveGetAccounts( fixtures.accountsPropertiesProfiles.accounts );
+				dispatch( STORE_NAME ).finishResolution( 'getAccounts', [] );
+
+				dispatch( STORE_NAME ).receiveGetProperties( [], { accountID } );
+				dispatch( STORE_NAME ).finishResolution( 'getProperties', [ accountID ] );
+			},
+		} );
+
+		expect( container ).toBeEmptyDOMElement();
+
+		await act( () => registry.dispatch( STORE_NAME ).setAccountID( ACCOUNT_CREATE ) );
```

Comment:
Once we set the accountID, we need to check that the PropertySelect element becomes visible.

Reply:
So (as per my last comment) I'm reusing the existing tests but changing them to expect hidden

The current tests go from visible to hidden but I'll also dispatch extra actions to check it toggles back to visible :+1: